ana_scans.py

```diff
@@ -83,19 +86,18 @@ def launchAnaArgs(anaType, cName, cType, scandate, scandatetrim=None, ztrim=4.0,
       cmd.append("--fileScurveFitTree=%s"%(filename_Trim))
       pass
 
-    postCmds[0] = ["mkdir","-p","%s"%(elogPath)]
     postCmds[1] = ["cp","%s/ThresholdScanData/ThreshSummary.png"%(dirPath),
                    "%s/ThreshSummary_%s.png"%(elogPath,cName)]
     postCmds[2] = ["cp","%s/ThresholdScanData/ThreshPrunedSummary.png"%(dirPath),
                    "%s/ThreshPrunedSummary_%s.png"%(elogPath,cName)]
-    postCmds[3] = ["cp","%s/ThresholdScanData/chConfig_MasksUpdated.txt"%(dirPath),
-                   "%s/chConfig_MasksUpdated_%s.txt"%(elogPath,cName)]
-    postCmds[4] = ["cp","%s/ThresholdScanData/vfatConfig.txt"%(dirPath),
+    postCmds[3] = ["cp","%s/ThresholdScanData/vfatConfig.txt"%(dirPath),
                    "%s/vfatConfig_%s.txt"%(elogPath,cName)]
```

Comment:
If we provide another environment variable, e.g. `$CONFIG_PATH` then maybe that's the solution.

Otherwise I think it would be convenient to have a single place to have all the files so the question:

"What are the thresholds set to" has an easy single entry point to access.

For now I would propose `$ELOG_PATH` and then in a later feature request we consider `$CONFIG_PATH`.